lib/param_parsing/toml.go

```diff
@@ -0,0 +1,32 @@
+package param_parsing
+
+import (
+	"fmt"
+	"github.com/spf13/viper"
+	"github.com/warrensbox/terraform-switcher/lib"
+	"log"
+)
+
+// GetParamsTOML parses everything in the toml file, return required version and bin path
+func GetParamsTOML(params Params) Params {
+	tomlFileName := ".tfswitch.toml"
+	tomlPath := params.ChDirPath + "/" + tomlFileName
```

Comment:
It probably makes sense to use more abstract var names like you used in other similar functions (`filePath` + `fileName`) and align those other functions to use the same approach (`filePath` + `fileName`) for consistency and simplicity.